src/scripts/clipperUI/componentBase.ts

```diff
@@ -121,6 +120,12 @@ export abstract class ComponentBase<TState, TProps> {
 			}
 		};
 
+		invokeAttributes.onkeydown = (e: KeyboardEvent) => {
+			if (e.which === decreaseButton || e.which === increaseButton) {
```

Comment:
If the user is a horizontal key navigatable widget and hits up/down, does the background page scroll?